addons/account/tests/test_reconciliation.py

```diff
@@ -195,23 +195,6 @@ def create_move(name, amount, amount_currency, currency_id):
             self.assertEqual(aml.amount_residual, 0, 'The journal item should be totally reconciled')
             self.assertEqual(aml.amount_residual_currency, 0, 'The journal item should be totally reconciled')
 
-        move_list_vals = [
```

Comment:
are they removed or moved elsewhere, untouched?

Reply:
No but this part doesn't make sense anymore since the tested extra parameters of `reconcile` are no longer supported:
`aml_recs.reconcile(self.account_rsa, self.bank_journal_usd)`

Reply:
that was used to reconcile everything while creating a writeoff entry. How do you manage that now?

Reply:
This was never used. The manual reconciliation widget is creating its own writeoff before calling reconcile.

Reply:
See: https://github.com/odoo/enterprise/blob/3a5ca791d3e8252e8d7d1012dc7553b29ef348c4/account_accountant/models/reconciliation_widget.py#L933

Reply:
ok, that was old code legacy (from the old manual reconciliation wizard) then. 

And I suppose that when you register a partial payment on an invoice and say you want it fully paid it also has its own writeoff creation code, mmh?

Reply:
indeed. This part is managed in account.payment